Geometry/HGCalCommonData/interface/HGCalProperty.h

```diff
@@ -5,10 +5,11 @@
 #include <cstdint>
 
 namespace HGCalProperty {
-  int32_t waferProperty(const int32_t thick, const int32_t partial, const int32_t orient);
+  int32_t waferProperty(const int32_t thick, const int32_t partial, const int32_t orient, const int32_t casette = 0);
```

Comment:
Did you maybe meant "cassette" instead of "casette"?
If so, better to fix the name, in order to make the code better understandable.